tests/test_commands.py

```diff
@@ -109,3 +109,24 @@ def test_enqueue_files(self):
         self.__send("enqueue-files "
                     "one,two\\, please,slash\\\\.mp3,four")
         self.assertEquals(app.window.playlist.q.get(), songs)
+
+    def test_with_pattern(self):
```

Comment:
I think it would be good to test the case of some valid JSON that's also a query

`"foo"`
(or even `1.0` etc)

Reply:
You're right.  This actually revealed a bug in the PR.